<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PresetButtonGroup</class>
 <widget class="QWidget" name="PresetButtonGroup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>96</width>
    <height>27</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>9</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="toolButton_play">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton
{
	border: 0px;
	border-image: url(:/ptz/ptz/patrolBegin_gray.png);
}
QToolButton:hover
{
	border-image: url(:/ptz/ptz/play_hover.png);
}
QToolButton:pressed
{
	border-image: url(:/ptz/ptz/patrolBegin_gray.png);
}
QToolButton:disabled
{
	border-image: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_save">
     <property name="styleSheet">
      <string notr="true">QToolButton
{
	border: 0px;
	border-image: url(:/ptz/ptz/save_gray.png);
}
QToolButton:hover
{
	border-image: url(:/ptz/ptz/save_hover.png);
}
QToolButton:pressed
{
	border-image: url(:/ptz/ptz/save_gray.png);
}
QToolButton:disabled
{
	border-image: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="toolButton_delete">
     <property name="styleSheet">
      <string notr="true">QToolButton
{
	border: 0px;
	border-image: url(:/ptz/ptz/patrolDelete_gray.png);
}
QToolButton:hover
{
	border-image: url(:/ptz/ptz/delete_hover.png);
}
QToolButton:pressed
{
	border-image: url(:/ptz/ptz/patrolDelete_gray.png);
}
QToolButton:disabled
{
	border-image: none;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
